dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/JaxRsIntegration.java

```diff
@@ -170,6 +172,7 @@ public static void setUp() throws Exception {
         ShrinkHelper.exportAppToServer(server, jaegerApp, SERVER_ONLY);
         ShrinkHelper.exportAppToServer(server, asyncServerApp, SERVER_ONLY);
         ShrinkHelper.exportAppToServer(server, methodsApp, SERVER_ONLY);
+        server.setJvmOptions(new ArrayList<>(Arrays.asList("-Dio.opentelemetry.context.enableStrictContext=true")));
```

Comment:
I don't think there's any need to create a new `ArrayList<>` here, you should just be able to pass in the result of `Arrays.asList` (or `Collections.singletonList`).